package.json

```diff
@@ -8,7 +8,7 @@
     "@graasp/query-client": "git://github.com/graasp/graasp-query-client.git#main",
     "@graasp/ui": "git://github.com/graasp/graasp-ui.git#41/rollup",
     "@material-ui/core": "4.11.2",
-    "@material-ui/icons": "4.11.2",
+    "@material-ui/icons": "5.0.0-beta.4",
```

Comment:
Why was this upgraded to beta?

Reply:
This was upgraded to have the pushPin Icon, it doesn't exists on 4.11.2